fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalLimitOperator.java

```diff
@@ -80,7 +80,7 @@ public int hashCode() {
 
     public static class Builder
             extends LogicalOperator.Builder<LogicalLimitOperator, LogicalLimitOperator.Builder> {
-        private long offset = -1;
+        private long offset = DEFAULT_OFFSET;
 
```

Comment:
DEFAULT_OFFSET is 0 but here is previous value is -1, is that right?

Reply:
No problem~